gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -171,46 +164,233 @@ def _get_prior_measurements_in_world_frame(
         Returns:
             BinaryMeasurementsPoint3 containing Point3 priors in world frame.
         """
-        if len(i2Ti1_priors) == 0:
-            return gtsam.BinaryMeasurementsPoint3()
 
         def get_prior_in_world_frame(i2, i2Ti1_prior):
             return wRi_list[i2].rotate(i2Ti1_prior.value.translation())
 
-        w_i2ti1_priors = gtsam.BinaryMeasurementsPoint3()
+        w_i1ti2_prior_measurements = BinaryMeasurementsPoint3()
+        if len(i2Ti1_priors) == 0:
+            return w_i1ti2_prior_measurements
 
+        # TODO(akshay-krishnan): Use the translation covariance, transform to world frame.
+        # noise_model = gtsam.noiseModel.Gaussian.Covariance(i2Ti1_prior.covariance)
+        # TODO(akshay-krishnan): use robust noise model for priors?
+        noise_model = gtsam.noiseModel.Isotropic.Sigma(3, 1e-2)
         for (i1, i2), i2Ti1_prior in i2Ti1_priors.items():
-            # TODO(akshay-krishnan): Use the translation covariance, transform to world frame.
-            # noise_model = gtsam.noiseModel.Gaussian.Covariance(i2Ti1_prior.covariance)
-            noise_model = gtsam.noiseModel.Isotropic.Sigma(3, 1e-2)
-            w_i2ti1_priors.append(
+            w_i1ti2_prior_measurements.append(
                 BinaryMeasurementPoint3(
-                    i2,
-                    i1,
+                    C(i2),
+                    C(i1),
                     get_prior_in_world_frame(i2, i2Ti1_prior),
                     noise_model,
                 )
             )
-        return w_i2ti1_priors
+        return w_i1ti2_prior_measurements
+
+    def compute_inliers(
+        self,
+        w_i2Ui1_dict: RelativeDirectionsDict,
+        w_i2Ui1_dict_tracks: RelativeDirectionsDict,
+    ) -> Tuple[RelativeDirectionsDict, RelativeDirectionsDict, Set[int]]:
+        """Perform inlier detection for the relative direction measurements.
+
+        Args:
+            w_i2Ui1_dict: Dictionary of Unit3 relative translations between cameras.
+            w_i2Ui1_dict_tracks: Dictionary of Unit3 relative translations between cameras and landmarks.
+
+        Returns:
+            Set of indices (i1, i2) which are inliers.
+        """
+
+        # Sample directions for projection
+        combined_measurements = list(w_i2Ui1_dict.values()) + list(w_i2Ui1_dict_tracks.values())
+        projection_directions = self.__sample_projection_directions(combined_measurements)
 
-    def __get_initial_values(self, wTi_initial: List[Optional[PosePrior]]):
+        # Convert to measurements - indexes to symbols.
```

Comment:
nit: what does the hyphen mean here? prefer something else since can be confused with minus

Reply:
Noting as resolved, thanks!